src/features/metadata-instance-editor/Instance.js

```diff
@@ -207,8 +208,11 @@ class Instance extends React.PureComponent<Props, State> {
             isDirty,
             isCascadingPolicyApplicable,
             onSave,
+            template,
         }: Props = this.props;
         const { data: currentData, errors, isCascadingEnabled, isCascadingOverwritten }: State = this.state;
+        const isCustomInstance = template.templateKey === TEMPLATE_CUSTOM_PROPERTIES;
+        const hasCascadePermission = isCustomInstance ? true : isCascadingPolicyApplicable;
```

Comment:
is this saying all custom metadata is cascaded?

is the cascading policy only for the metadata templates created in admin console?

Reply:
_is this saying all custom metadata has permission to cascade? (is the same true for metadata on files as well?)_
Cascading metadata policies cannot be applied to custom metadata, so we do not utilize cascade permissions when determining a user's ability to modify and save a custom template. Admittedly, the variable name here lacks clarity.